### PR TITLE
Renamed logo fetcher urls

### PR DESCRIPTION
**Pull request by spiderjako**
#### Summary
 * Added a migration to rename the logo64_url to just logo_url, thus removing logo_url beforehand, combined with logo32_url's removal

### CODE REVIEW

File: lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex [hunk 1]
```diff
@@ -21,15 +21,14 @@ defmodule Sanbase.ExternalServices.Coinmarketcap.LogoFetcher do
 
     Temp.track!()
     dir_path_64 = Temp.mkdir!("logos_64")
```
Comment (kalkov) [inline lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex]: As there is only one size now, you can remove the `_64` postfix from the names

File: lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex [hunk 2]
```diff
@@ -116,16 +112,10 @@ defmodule Sanbase.ExternalServices.Coinmarketcap.LogoFetcher do
     end
   end
 
-  defp resize_image(source_filepath, dest_dir_path, filename) do
-    dest_filepath = dest_dir_path <> "/" <> filename
-    Mogrify.open(source_filepath) |> Mogrify.resize("32x32") |> Mogrify.save(path: dest_filepath)
```
Comment (kalkov) [inline lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex]: Remove the `mogrify` dependency from the mix.exs

Comment (kalkov) [reply lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex]: Also `imagemagick` from Dockerfiles

File: priv/repo/migrations/20190828092907_remove_logo_url.exs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+defmodule Sanbase.Repo.Migrations.RemoveLogoUrl do
```
Comment (kalkov) [inline priv/repo/migrations/20190828092907_remove_logo_url.exs]: Create 3 migrations in that order:
1. Remove `logo_url`
2. Rename `logo64_url` to `logo_url`
3. Remove `logo32_url`

File: lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex (hunk 2 above)
Comment (spiderjako) [reply lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex]: Yes, I was going to ask if I should remove the resizing, when there is no 32x32 anymore

File: lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex (hunk 1 above)
Comment (spiderjako) [reply lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex]: You're right

File: priv/repo/migrations/20190828092907_remove_logo_url.exs (hunk 3 above)
Comment (spiderjako) [reply priv/repo/migrations/20190828092907_remove_logo_url.exs]: It shall be done.

File: lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex (hunk 2 above)
Comment (kalkov) [reply lib/sanbase/external_services/coinmarketcap2/logo_fetcher.ex]: Actually I changed my mind, let's keep the resize but change it to 64x64. If CMC changes the size of the logos or a single logo is with different size that can break things on our side and we will need to change the script. Having a resize step will guarantee us that the logos will always have the same size.

File: priv/repo/migrations/20190828104909_remove_logo_32_url.exs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+defmodule Sanbase.Repo.Migrations.RemoveLogo32Url do
+  use Ecto.Migration
+
+  def change do
```
Comment (kalkov) [inline priv/repo/migrations/20190828104909_remove_logo_32_url.exs]: I don't know about the others but I like to have reversible migrations. Try `mix ecto.rollback --step NUMBER` and you will see it fails. 
Ecto 3 has [`remove/3`](https://hexdocs.pm/ecto_sql/Ecto.Migration.html#remove/3) which is reversible, but we are still not there.
The way to do it in Ecto 2 is with `up/down`:
```elixir
  def up do
    alter table("project") do
      remove(:logo32_url)
    end
  end

  def down do
    alter table("project") do
      add(:logo32_url, :string)
    end
  end
```

File: priv/repo/migrations/20190828104846_rename_logo_64_url_to_logo_url.exs [hunk 5]
```diff
@@ -0,0 +1,7 @@
+defmodule Sanbase.Repo.Migrations.RenameLogo64UrlToLogoUrl do
+  use Ecto.Migration
+
+  def change do
+    rename(table("project"), :logo64_url, to: :logo_url)
```
Comment (kalkov) [inline priv/repo/migrations/20190828104846_rename_logo_64_url_to_logo_url.exs]: @valo Is this migration reversible?

Comment (valo) [reply priv/repo/migrations/20190828104846_rename_logo_64_url_to_logo_url.exs]: Should be. Have you tested it?

Comment (kalkov) [reply priv/repo/migrations/20190828104846_rename_logo_64_url_to_logo_url.exs]: Yes it is, tested it.

**Pull request merged**